python/packages/nisar/workflows/gen_el_null_range_product.py

```diff
@@ -209,6 +218,29 @@ def gen_el_null_range_product(args):
         raw_obj, args.freq_band, args.txrx_pol)
     logger.info(f'List of selected frequency bands and TxRx Pols -> {frq_pol}')
 
+    # determine [start, stop] range lines of echo to be processed.
+    freq_band = list(frq_pol.keys())[0]
+    txrx_pol = frq_pol[freq_band][0]
+    epoch, tm = raw_obj.getPulseTimes(freq_band, txrx_pol[0])
```

Comment:
`epoch` seems to be unused

```suggestion
    _, tm = raw_obj.getPulseTimes(freq_band, txrx_pol[0])
```